test/serialize-deserialize-test.coffee

```diff
@@ -0,0 +1,170 @@
+{assert} = require('chai')
+sinon = require('sinon')
+
+blueprintApi = require('../src/blueprint-api')
+
+
+describe('Serialization of the Blueprint interface (Application AST)', ->
+  describe('Blueprint object', ->
```

Comment:
example call in Apiary:

`blueprintApi.Blueprint.fromJSON(astCache)`
`new blueprintApi.Resource {...data...}`
`new blueprintApi.Response {...data...}`

so that is why I really want these tests to remain here, since those are the only description of Apiary-Application AST for now.